dask/dataframe/io/parquet.py

```diff
@@ -13,41 +14,35 @@
 from ...delayed import delayed
 from ...bytes.core import get_fs_paths_myopen
 
-try:
-    import fastparquet
-    from fastparquet.core import read_row_group_file
-    from fastparquet.api import _pre_allocate
-    from fastparquet.util import check_column_names
-except ImportError:
-    fastparquet = False
-else:
-    @normalize_token.register(fastparquet.ParquetFile)
-    def normalize_ParquetFile(pf):
-        return (type(pf), pf.fn, pf.sep) + normalize_token(pf.open)
-
-try:
-    import pyarrow.parquet as pyarrow_parquet
-except ImportError:
-    pyarrow_parquet = False
-else:
-    @normalize_token.register(pyarrow_parquet.ParquetDataset)
-    def normalize_PyArrowParquetDataset(ds):
-        return (type(ds), ds.paths)
-
-
-def _meta_from_dtypes(to_read_columns, file_columns, file_dtypes):
+
+__all__ = ('read_parquet', 'to_parquet')
+
+
+def _meta_from_dtypes(to_read_columns, file_columns, file_dtypes, index_cols):
```

Comment:
What kind of support is there in `fastparquet` for the following pandas use cases?

1. MultiIndex columns and indexes
1. Columns whose name is the same as an index level

Reply:
@martindurant , do you know the answer to this?

Reply:
1. No support for multi-index. It would not be too hard to add.
2. I don't know, I don't imagine it's a problem, but there is no test.